src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -59,10 +59,13 @@
   "minecolonies.config.listofplantables.comment": "The items and item-tags that the Florist can plant.",
 
   "item.minecolonies.scepterlumberjack": "Forest Tool",
-  "item.minecolonies.scepterlumberjack.usedstart": "First position set.",
-  "item.minecolonies.scepterlumberjack.usedend": "Second position set.",
-  "item.minecolonies.scepterlumberjack.restrictiontoobig": "Area %s is too large. Maximum restricted area for the Forester is %s blocks.",
-  "item.minecolonies.scepterlumberjack.restrictionset": "Forester restricted area set. Size is %s blocks. Max size is %s blocks.",
+  "item.minecolonies.scepterlumberjack.usedstart": "First position set. (Click same position to finish)",
```

```suggestion
  "item.minecolonies.scepterlumberjack.usedstart": "First position set. Click the same position to finish.",
```
For the second position as well